README.md

```diff
@@ -1,3 +1,75 @@
-# opensource
+<div align="center">
+  <a href="#">
```

Comment:
why are we putting images inside a tag?

Reply:
Because we are using HTML to align images to the center.

Reply:
href aru kei rakhne ki? Image ma click garda you scroll to the top. very alter